.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This should match the team set up in https://github.com/orgs/opensearch-project/teams and include any additional contributors
-*   @opensearch-project/clients @axeoman @deztructor
\ No newline at end of file
+*   @VachaShah @dblock @harshavamsi @axeoman @deztructor @rushiagr @Shephalimittal
```

Comment:
Weird, `Unknown owner` complains about @harshavamsi and @rushiagr. Could you please double-check the rights?

Reply:
I fixed @harshavamsi by adding him to permissions. I believe @rushiagr should be moved to alumni?